index.html

```diff
@@ -11,23 +11,23 @@
   <body>
     <div class="container">
       <header class="" id="header">
-        <h1><span class="lang_pl">WISIELEC</span><span class="" id="slash"> / </span><span class="lang_uk">HANGMAN</span></h1>
+        <h1><span class="pl_pl">WISIELEC</span><span class="" id="slash"> / </span><span class="en_gb">HANGMAN</span></h1>
         <p>Wybierz język / Choose language</p>
         <div class="flags">
-          <a href="#" class="flag_pol" id="flag_pol"><img src="pics/pol.GIF" alt="poland flag"></a>
-          <a href="#" class="flag_uk" id="flag_uk"><img src="pics/uk.GIF" alt="uk flag"></a>
+          <a href="#" class="flag_pl" id="flag_pl"><img src="pics/pl.GIF" alt="poland flag"></a>
+          <a href="#" class="flag_gb" id="flag_gb"><img src="pics/gb.GIF" alt="great britain flag"></a>
         </div>
       </header>
       <div class="opacityZero hidden" id="board">
```

Comment:
Code standards (all classes should be written with "-" between words) 
e.g class="opacity-zero"